src/Mailer/Transport/DebugKitTransport.php

```diff
@@ -91,7 +93,10 @@ public function send(Message $message): array
      */
     public function __call(string $method, array $args): mixed
     {
-        return call_user_func_array([$this->originalTransport, $method], $args);
+        /** @var callable $callable */
+        $callable = [$this->originalTransport, $method];
+
+        return call_user_func_array($callable, $args);
```

Comment:
We could replace this with a first-class callable. $this->originalTransport->$method(...).